tests/rafter/Gopkg.toml

```diff
@@ -0,0 +1,36 @@
+required = [
+    "golang.org/x/tools/cmd/goimports",
+    "github.com/kisielk/errcheck",
+]
+
```

Comment:
I've deleted 
```
[[constraint]]
  name="github.com/kyma-project/kyma"
  revision = "f5d791b218819a0aecde663dedda1905d7c12bb5"
```
section from assetstore, so that it compiles - do I add it back?

Reply:
In next PR or maybe just leave this as it is?

Reply:
If you don't use in code imports from kyma (probably we only used it in old implementation for asset store resources), feel free to remove it.